docs/components/mappers/modbus_mapper_v2.md

```diff
@@ -0,0 +1,76 @@
+# Modbus Mapper V2
+
+# Introduction
+The Modbus mapper is an interface between KubeEdge and Modbus devices. It could set/get device data, get and report the device status.
+In this version, Modbus TCP and RTU mode are supported.
+ 
+## Running the mapper
+
+  1. Please ensure that the Modbus device is connected to your edge node
+  2. Set 'modbus=true' label for the node (This label is a prerequisite for the scheduler to schedule modbus_mapper pod on the node)
+
+      ```shell
+      kubectl label nodes <name-of-node> modbus=true
+      ```
+
+  3. Build and deploy the mapper by following the steps given below.
+
+### Building the Modbus mapper
+
+ ```shell
+cd $GOPATH/src/github.com/kubeedge/kubeedge/
+make modbusmapper or `make mappers` if you want to build all mappers
+docker tag modbusmapper:v1.0 <your_dockerhub_username>/modbusmapper:v1.0
+docker push <your_dockerhub_username>/modbusmapper:v1.0
+
+Note: Before trying to push the docker image to the remote repository please ensure that you have signed into docker from your node, if not please type the following command to sign in
+ docker login
+ # Please enter your username and password when prompted
+ ```
+
+### Deploying Modbus mapper application
+
+```shell
+cd $GOPATH/src/github.com/kubeedge/kubeedge/mappers/modbus-go
+
+# Please enter the following details in the deployment.yaml :-
+#    1. Replace <edge_node_name> with the name of your edge node at spec.template.spec.voluems.configMap.name
+#    2. Replace <your_dockerhub_username> with your dockerhub username at spec.template.spec.containers.image
+#    3. If the Modbus device is a serial device, replace <hostPath> with the device name at spec.template.voluems.hostPath.path
+
+kubectl create -f deployment.yaml
+```
+
+## Modules
+
+The Modbus mapper consists of the following major modules :-
+
+ 1. Flags
+ 2. Configmap parser
+ 3. Driver
+ 4. Device
+ 5. Event process
+ 6. Timer
+
+ ### Flags
+ The default configuration file is config.yaml, you could specify it by `--config-file`.
+ The `--v` flag could specify the log level. The level definition is as klog.
+ The configurations include MQTT user/password/Certification.
+
+ ### Configmap parser
+ The configmap is generated by the device controller after you create a device model and device instance. It's mounted at /opt/kubeedge/deviceProfile.json
+ at run time.
```

Comment:
Add some tips about if user change device model or instance when modbus mapper is running?

Reply:
Done. Thanks for the suggestion.